exceptions.py

```diff
@@ -154,10 +154,6 @@ class UnquotedAssignTextException(HedyException):
     def __init__(self, text):
         super().__init__('Unquoted Assignment', text=text)
 
-class EmptyProgramException(HedyException):
```

Comment:
I am not sure why we are removing this exception here, how does that relate to the tests?

Reply:
This is just cleaning dead code. The exception is not used. @TiBiBa commented out the exception throwing when introducing babel. I also confirmed with him that this exception is not used and not needed. Would you like to reintroduce it?

Reply:
Yes this exception has become redundant when introducing the inline comments. If I remember correctly @jpelay removed the throwing of this error when working on fixing the failing tests of the comments.

Reply:
sorry, I just realized I did not answered the exact question. I removed this exception together with its tests which were commented out. I only noticed it is dead code because of cleaning the tests. I could also remove the exception in a separate PR.

Reply:
Yes this exception is no longer used. If we include it, then programs that consist exclusevely of comments (of which the adventures have a lot) will fail, because since lark will ignore those tokens, to our transpiler it will be as if they're empty programs.